ethpm/backends/registry.py

```diff
@@ -2,7 +2,8 @@
 from urllib import parse
 
 from eth_utils import to_bytes
-from web3.auto.infura import w3
+from web3 import Web3
+from web3.providers.auto import load_provider_from_uri
 
 from ethpm.backends.base import BaseURIBackend
 from ethpm.constants import INFURA_API_KEY
```

Comment:
Can we get a tracking issue for this?  We should probably plan to be rid of it within a month or two at the very most.

Reply:
Just in terms of removing the `INFURA_API_KEY` from `constants.py` and requiring users to provide their own?

Reply:
Removing it from constants and coming up with the mechanism that 1) doesn't require users to use infura but 2) still preserves as much ease of usability as possible.  

This probably really means leaving the `ethpm` library such that it require *more* manual configuration and addressing the usability component in the web3.py codebase.